docs/en/stack/ml/anomaly-detection/ml-limitations.asciidoc

```diff
@@ -70,6 +70,31 @@ You cannot use the following field names in the `by_field_name` or
 `over_field_name` properties in a job: `by`; `count`; `over`. This limitation
 also applies to those properties when you create advanced jobs in {kib}.
 
+
+[discrete]
+[[ml-arrays-limitations]]
+=== Arrays in analyzed fields are turned into comma-separated strings
+
+If an {anomaly-job} is configured to analyze an aggregatable field (a field that 
+is part of the index mapping definition), and this field contains an array, then 
+the array is turned into a comma-separated concatenated string. The items in the 
+array are sorted alphabetically and the duplicated items are removed. For 
+example, the array `["zebra", "dog", "cat", "alligator", "cat"]` becomes 
+`alligator,cat,dog,zebra`. The Anomaly Explorer charts don't display results as 
```

Comment:
> The Anomaly Explorer charts don't display results as 

Does it only omit results related to this field (e.g. in the Anomalies table) or does its presence affect the ability to render *all* results?

Reply:
@lcawl Addressed via 5b7954c
I tested it, the charts are blank.